packages/web/components/complex/pools-table.tsx

```diff
@@ -111,11 +111,7 @@ export const PoolsTable = (props: PropsWithChildren<PoolsTableProps>) => {
     children,
   } = props;
 
-  /** Won't sort when searching is happening. */
-  const sortKey = useMemo(
-    () => (Boolean(filters.searchQuery) ? undefined : sortParams.allPoolsSort),
-    [filters.searchQuery, sortParams.allPoolsSort]
-  );
+  const sortKey = sortParams.allPoolsSort;
```

Comment:
Even though the tRPC procedures supports searching and sorting at the same time, I think it could yield confusing results, especially with lower cap tokens. 

Just for this table. I think it's OK to sort + search with the few items on the token page

I was getting complaints about this when I implemented this new pools table

However, I was also getting complaints about not being able to sort + search. But, we agreed that when first searching the sort would be disabled. Then after the search they could sort if they want. That would help make it clearer that searching + sorting is happening at the same time. I'm not saying you should do this, it's just some background on what we haven't gotten to doing yet

Reply:
Ye makes sense, I'll add a different field for search by denoms because it's a different use case if compared to search + sort

Reply:
Is that necessary? What about just not passing sort params to the tRPC procedure while the user is sorting? Or WDYM by field?

Reply:
I need it because inside the token info page we need pools filtered by denom but also sorted by volume not by liquidity